apps/catalogue/pages/[schema]/catalogue/[catalogue]/[resourceType]/[resource]/index.vue

```diff
@@ -5,12 +5,16 @@ import collectionEventsQuery from "../../../../../../gql/collectionEvents";
 import datasetQuery from "../../../../../../gql/datasets";
 import ontologyFragment from "../../../../../../gql/fragments/ontology";
 import fileFragment from "../../../../../../gql/fragments/file";
+import variablesQuery from "../../../../../../gql/variables";
```

Comment:
I thought '~/gql/datasets' would also work? Or is there no way to tell build about the root?
Maybe that goes wrong when there are cross package imports?

Reply:
ah, we need absolute paths somehow. And if we would use 'imports' in the package.json? 
What is wrong with that? Is the issue that the treeshake is not good enough and pages get bloathed?